fe/fe-core/src/main/java/com/starrocks/catalog/MaterializedView.java

```diff
@@ -548,18 +551,17 @@ public Expr getFirstPartitionRefTableExpr() {
     }
 
     public Set<String> getUpdatedPartitionNamesOfOlapTable(OlapTable baseTable) {
+        // Ignore partitions when mv 's last refreshed time period is less than `maxMVRewriteStaleness`
+        if (isStalenessSatisfy()) {
```

Comment:
`isStalenessSatisfied` , or `satisfyStaleness` ?

Reply:
if (!satisfyStaleness) { return empty } ?

Reply:
> if (!satisfyStaleness) { return empty } ?

`getUpdatedPartitionNamesOfOlapTable ` needs updated partitions' names, just return empty if staleness is satisfied and no need check anymore.